custom_components/jcihitachi_tw/climate.py

```diff
@@ -108,6 +109,20 @@ def target_temperature(self):
         """Return the target temperature."""
         status = self.hass.data[DOMAIN][UPDATED_DATA][self._thing.name]
         if status:
+            if status.target_temp == 65535:
+                if status.mode == "fan":
+                    if self._prev_target == -1:
+                        self._prev_target = status.min_temp
+                    _LOGGER.debug(f"no target temp defined in fan mode, returning previous target: {self._prev_target}")
+                    return self._prev_target
+                elif status.mode == "auto":
+                    if self._prev_target == -1:
+                        self._prev_target = status.min_temp
+                    _LOGGER.debug(f"no target temp defined in auto mode, returning previous target: {self._prev_target}")
+                    return self._prev_target
```

Comment:
We can combine `fan` and `auto` conditions. `status.mode` can be filled into the formatted debugging string.

Reply:
Sure, I had simplified those lines as shown in
https://github.com/qqaatw/JciHitachiHA/pull/36/commits/2ab200c5aa929c4a0639778b783cb5d595add5d5#diff-259fd1253c24359a733707422979fff02361054aefafbe0fde15e5ffded27930R113-R114